theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -1,3 +1,22 @@
+def railsEnvForTask(task, env) {
+    if (task == 'assets:precompile') {
+        return 'nulldb://nohost'
+    } else {
+        // TODO: limit DB name length to < 63 chars
+        ruby = env.RUBY_VER.split('.')[0..1]
+        task_id = task.split(':')[1]
+        database = (['test', env.JOB_NAME, env.BUILD_ID] + ruby + [task_id]).join('-')
+        if (database.size() > 63) {
+            error "${id} cannot be used to generate a PostgreSQL DB name, the resulting name would be longer than 63 chars."
+        }
+        return "postgresql://foreman:foreman@localhost/${database}"
+    }
+}
+
+def databaseUrlForTask(task) {
+    return task == 'assets:precompile' ? 'production' : 'test'
+}
+
 def databaseFile(id) {
```

Comment:
is this still used somewhere then?

Reply:
https://github.com/theforeman/jenkins-jobs/blob/e4c55790f55bfc196a5b7eda5460b7e20ce37818/theforeman.org/pipelines/release/source/katello.groovy#L30
Do you want me to also replace that?

Reply:
`theforeman.org/pipelines/release/source/katello.groovy` does use this, but I guess it'd get the same treatment once this PR is done and we see it working fine

Reply:
Unrelated, but I had a closer look at the other functions and noticed `addSettings` was also unused: https://github.com/theforeman/jenkins-jobs/pull/524